<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="3" passed="1">
  <reporter-output>
    <line>
      <![CDATA[Soft Assert]]>
    </line>
    <line>
      <![CDATA[<br />onTestFail<hr />]]>
    </line>
    <line>
      <![CDATA[Report log++++++++++++++++++++++++++<br />]]>
    </line>
    <line>
      <![CDATA[Report log++++++++++++++++++++++++++]]>
    </line>
    <line>
      <![CDATA[level0]]>
    </line>
    <line>
      <![CDATA[<br />onTestFail<hr />]]>
    </line>
    <line>
      <![CDATA[<a href='http://www.google.com'>google</a>]]>
    </line>
    <line>
      <![CDATA[<br />onTestSuccess<hr /]]>
    </line>
  </reporter-output>
  <suite name="MySuite" duration-ms="56" started-at="2017-03-17T07:55:23Z" finished-at="2017-03-17T07:55:23Z">
    <groups>
      <group name="setup">
        <method signature="DependencyTest.configTest()[pri:0, instance:com.example.DependencyTest@36f6e879]" name="configTest" class="com.example.DependencyTest"/>
        <method signature="DependencyTest.connectionTest()[pri:0, instance:com.example.DependencyTest@36f6e879]" name="connectionTest" class="com.example.DependencyTest"/>
      </group> <!-- setup -->
    </groups>
    <test name="dependencyTest" duration-ms="56" started-at="2017-03-17T07:55:23Z" finished-at="2017-03-17T07:55:23Z">
      <class name="com.example.DependencyTest">
        <test-method status="PASS" signature="setup(org.testng.ITestContext)[pri:0, instance:com.example.DependencyTest@36f6e879]" name="setup" is-config="true" duration-ms="7" started-at="2017-03-17T15:55:23Z" finished-at="2017-03-17T15:55:23Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@6e06451e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="configTest()[pri:0, instance:com.example.DependencyTest@36f6e879]" name="configTest" duration-ms="4" started-at="2017-03-17T15:55:23Z" finished-at="2017-03-17T15:55:23Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
null, ooooooooooooooookoooo]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
null, ooooooooooooooookoooo
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:37)
	at com.example.DependencyTest.configTest(DependencyTest.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:74)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Soft Assert]]>
            </line>
            <line>
              <![CDATA[<br />onTestFail<hr />]]>
            </line>
          </reporter-output>
        </test-method> <!-- configTest -->
        <test-method status="FAIL" signature="connectionTest()[pri:0, instance:com.example.DependencyTest@36f6e879]" name="connectionTest" duration-ms="1" started-at="2017-03-17T15:55:23Z" finished-at="2017-03-17T15:55:23Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Missed! expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Missed! expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.example.DependencyTest.connectionTest(DependencyTest.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:74)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Report log++++++++++++++++++++++++++<br />]]>
            </line>
            <line>
              <![CDATA[Report log++++++++++++++++++++++++++]]>
            </line>
            <line>
              <![CDATA[level0]]>
            </line>
            <line>
              <![CDATA[<br />onTestFail<hr />]]>
            </line>
          </reporter-output>
        </test-method> <!-- connectionTest -->
        <test-method status="PASS" signature="query()[pri:0, instance:com.example.DependencyTest@36f6e879]" name="query" duration-ms="1" depends-on-groups="setup" started-at="2017-03-17T15:55:23Z" finished-at="2017-03-17T15:55:23Z">
          <reporter-output>
            <line>
              <![CDATA[<a href='http://www.google.com'>google</a>]]>
            </line>
            <line>
              <![CDATA[<br />onTestSuccess<hr /]]>
            </line>
          </reporter-output>
        </test-method> <!-- query -->
        <test-method status="PASS" signature="teatdown()[pri:0, instance:com.example.DependencyTest@36f6e879]" name="teatdown" is-config="true" duration-ms="0" started-at="2017-03-17T15:55:23Z" finished-at="2017-03-17T15:55:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teatdown -->
      </class> <!-- com.example.DependencyTest -->
    </test> <!-- dependencyTest -->
  </suite> <!-- MySuite -->
</testng-results>
